From 7aec62ed5d84700c86121261961da07fc7edd07c Mon Sep 17 00:00:00 2001
From: Stefan Becker <chemobejk@gmail.com>
Date: Sun, 1 Sep 2019 01:08:02 +0300
Subject: [PATCH 15/16] telsu: update time parser for new format

The time format is now 'HH.MM' instead of 'HH:MM'.
---
 grab/fi/fi/source/telsu.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grab/fi/fi/source/telsu.pm b/grab/fi/fi/source/telsu.pm
index eac8b988..d77303eb 100644
--- a/grab/fi/fi/source/telsu.pm
+++ b/grab/fi/fi/source/telsu.pm
@@ -116,7 +116,7 @@ sub grab {
 
 	if ($title && $time && $desc) {
 	  if (my($new, $start_h, $start_m, $end_h, $end_m) =
-	      $time->as_text() =~ /^(.+)\s(\d{2}):(\d{2})\s-\s(\d{2}):(\d{2})/) {
+	      $time->as_text() =~ /^(.+)\s(\d{2})[:.](\d{2})\s-\s(\d{2})[:.](\d{2})/) {
 	    $title = $title->as_text();
 	    $desc  = $desc->as_text();
 
-- 
2.21.0

